cypress/cypress/e2e/F-live-quiz-workflow.cy.ts

```
@@ -1190,25 +1190,37 @@ describe('Different live-quiz workflows', () => {
+    cy.deleteElement({ elementName: NRQuestion2Title })
+    cy.deleteElement({ elementName: FTQuestion2Title })
+    cy.deleteElement({ elementName: SEQuestion2Title })
+  })
+
+  it('Cleanup: Delete the created answer collection', () => {
+    cy.loginLecturer()
+    cy.get('[data-cy="resources"]').click()
+    cy.deleteAnswerCollection({ collectionName: SECollection })
+  })
+
+  it.only('Cleanup: Verify that all answer collections have been deleted properly', () => {
```

Biome (1.9.4)

[error] Don't focus the test.

The 'only' method is often used for debugging or during implementation. It should be removed before deploying to production.
Consider removing 'only' to ensure all tests are executed.
Unsafe fix: Remove focus from test.


(lint/suspicious/noFocusedTests)